meta-facebook/aalc-rpu/src/platform/plat_fsc.c

```diff
@@ -159,6 +160,7 @@ static uint8_t calculatePID(zone_cfg *zone_p, uint8_t *duty)
 
 		int16_t temp = (int16_t)tmp;
 
+		/*
```

Comment:
pls delete commented code

Reply:
delete in https://github.com/facebook/OpenBIC/commit/32fa14b03a5e6c38582ff51d0685ecefc98cf37f